conan/tools/meson/toolchain.py

```diff
@@ -49,33 +48,76 @@ class MesonToolchain(object):
     {% endfor %}
 
     [binaries]
-    {% if c %}c = {{c}}{% endif %}
-    {% if cpp %}cpp = {{cpp}}{% endif %}
-    {% if ld %}ld = {{ld}}{% endif %}
+    {% if c %}
+    c = {{c}}
+    {% endif %}
+    {% if cpp %}
+    cpp = {{cpp}}
```

Comment:
This diff makes difficult to see if something changed, or is it just new formatting?